assets/cohere.yaml

```diff
@@ -145,7 +145,26 @@
       support at cohere.ai.
 
 - type: model
-  name: Cohere Representation model
+  name: Command
```

Comment:
Minor: Given Command is such a common word, Cohere Command seems better to me, since the consumers of this graph include folks like policymakers who won't know Command is even a model/Cohere's _a priori_.